### ISSUE TITLE
about training V2.yaml

### ISSUE THREAD

**Issue by ksh11023** [Cc-Hy/CMKD#15]
Hello,

When training with V2.yaml file,  in train_cmkd.py why do you load pretrained_lidar_model to model.model_img?

![11](https://user-images.githubusercontent.com/50094436/214813495-058c473a-84ea-4947-ad40-7f833084c9c3.png)

Thank you.

**Comment by Cc-Hy** [comment]
Hi,
In the student model and the teacher model, there are several shared structures like BEV backbone and detection head.
We load the pre-trained weights from the teacher model to accerate convergence and potentially improve the performance of the student model.

![image](https://user-images.githubusercontent.com/82150240/214859234-06fb7c97-b557-48e9-9dac-28cb4a07a8f0.png)

**Comment by ksh11023** [comment]
Thank you for the swift reply!

I have one more question.
 When training 60 epochs(BEV DA), is it better to train without rpn loss? the code only train with depth loss and BEV loss.

Thank you.

**Comment by Cc-Hy** [comment]
Hi,
Yes, this can be considered as a trick in the training process, because when the BEV features are not correct, calculating RPN losses  is meaningless.
And similar technique has been used in other methods, for example, using only 2D detection losses first and then adding 3D detection losses.

**Comment by xiaoxusanheyi** [comment]
> 这可以被认为是训练过程中的一个技巧，因为当 BEV 特征不正确时，计算 RPN 损失是没有意义的。
> 其他方法也使用了类似的技术，例如，首先仅使用 2D 检测损失，然后添加 3D 检测损失。



> 感谢您的快速回复！
> 
> 我还有一个问题。 训练60个epochs(BEV DA)时，没有rpn loss训练是不是更好？该代号只使用深度损失和BEV损失进行训练。
> 
> 谢谢。

BEV.yaml训练是60个epoch，而文章还说后20个epoch是用V2.yaml训练的吗，但是只有10个epoch呀
